src/lib/plugin_apis/fs.api

```diff
@@ -2540,13 +2540,17 @@ typedef enum {
  * @fsck: support for fsck operations (check and repair)
  * @configure: support for changing properties of an existing filesystem
  * @features: other supported features
+ * @partition_id: partition ID used for this filesystem on MSDOS partitions
+ * @partition_type: partition type/GUID used for this filesystem GPT partitions
```

Comment:
Hmm, this is misleading on a first look. Any reason not to use something like `partition_id_mbr` and `partition_uuid_gpt`?

Reply:
`id` and `type` are "official" names for the entries in MSDOS and GPT partition tables, our part plugin also calls those like that. It might be confusing, but it's an advanced API, I'd expect users to know what they are doing.

Reply:
Alright, fair enough.